src/main/resources/io/jenkins/plugins/opentelemetry/job/MonitoringAction/jobMain.jelly

```diff
@@ -2,12 +2,18 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     <table style="margin-top: 1em; margin-left:1em;">
-        <t:summary icon="/plugin/opentelemetry/images/48x48/opentelemetry.png">
-            OpenTelemetry:
-            <j:forEach items="${it.links}" var="link" varStatus="loop">
-                <img src="${resURL}${link.iconUrl}" height="24px"/><a href="${link.url}" target="_blank">${link.label}</a>
-                <j:if test="${!loop.last}">, </j:if>
-            </j:forEach>
+        <t:summary icon="icon-otel icon-xlg">
+            OpenTelemetry
+            <ul class="jenkins-!-padding-left-0">
```

Comment:
this class will only work on somewhat recent versions of core, but it looks okay without this just improves it